Session/Home/HomeViewModel.swift

```diff
@@ -543,6 +578,7 @@ public class HomeViewModel: NavigatableStateHolder {
         DispatchQueue.main.asyncAfter(deadline: .now() + .seconds(1)) { [self, dependencies] in
             // Set flag that review prompt was already presented
             dependencies[defaults: .standard, key: .didShowAppReviewPrompt] = true
+            dependencies[defaults: .standard, key: .didIgnoreAppReviewPrompt] = true
```

Comment:
So I feel like this flag is a little backwards (ie. setting `didIgnoreAppReviewPrompt` when the prompt appears isn't entirely accurate as the user hasn't had the chance to actually see the prompt yet)

I wonder whether instead we should have a `didActionAppReviewPrompt` which gets set to `true` when the users presses one of the buttons (should it also be set when pressing the `X`?) and set to `false` prior to displaying the prompt - that way if we tie any future logic to this flag it is being set at more intuitive times